sim_3d_volumes.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun  8 02:07:02 2021
+
+@author: NLEGENDN
+"""
+import coords
+import pandas as pd
+import numpy as np
+import torch
+import os
+
+CUDA = torch.cuda.is_available()
+
+if CUDA:
+    Main_dir = "/scratch/ex-kdd-1/NicolasLegendre/Cryo/Data"
+else:
+    Main_dir = os.getcwd()+"\\Data\\"
+
+Particules = np.asarray([[1, 0, 0, 3], [0, 8, 0, 5], [0, 0, 7, 9]])
+N_particules = 2000
+Vol_size = 64
+Name = '4_points_3D'
+Long = 2
+
+
+def dist(point1, point2):
+    """
+
+    Parameters
+    ----------
+    point1 : tuple float.
+    point2 : tuple float.
+
+    Returns
+    -------
+    distance between point1 and point2.
+
+    """
+    a, b, c = point1
+    d, e, f = point2
+    return (a-d)**2+(b-e)**2+(c-f)**2
```

Comment:
how about: `np.linalg.norm(point1-point2)`? no need to reimplement the euclidean dist in a new function IMO